src/ChocoCCM.psd1

```diff
@@ -82,7 +82,6 @@
         'Remove-CCMGroupMember'
         'Get-CCMDeployment'
         'Get-CCMDeploymentStep'
-        'Get-CCMDeploymentStepResult'
```

Comment:
Why was this command export removed?

Reply:
Because the command doesn't exist, as far as I can tell.